src/client/actions/edit/index.js

```diff
@@ -0,0 +1,13 @@
+
+import { extend } from 'lodash'
+import { actions as articleActions } from './articleActions'
+import { actions as editActions } from './editActions'
+import { actions as errorActions } from './errorActions'
+import { actions as sectionActions } from './sectionActions'
+
+export const actions = extend({},
+  articleActions,
+  editActions,
+  errorActions,
+  sectionActions
+)
```

Comment:
Can just do 

```
export const actions = {
   ... 
}
```
since there's no need to merge with another object (only using `{}`). Using `extend` will create an intermediate object in memory and copy over by reference, creating a bit of indirection.